recipes/pynnotator/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pynnotator" %}
+{% set version = "1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4e777813fec487e4352ad4fd6bd55ae5286a120fab848cdba7c52c126645aab5"
+
+build:
+  number: 0
+  entry_points:
+    - pynnotator=pynnotator.main:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python 
+
+requirements:
+  host:
+    - Cython
+    - pip
+    - pysam
```

Comment:
Does the cython code compile against pysam or is this just used as a regular python module? In the latter case you can remove it from the `host` section.

Reply:
Just agains pysam. Removed from host section.